### PR TITLE
Added documentation for --sort option in translation:update command

### PR DESCRIPTION
**Pull request by k0d3r1s**
Documentation for symfony/symfony#33117

### CODE REVIEW

File: translation.rst [hunk 1]
```diff
@@ -264,6 +264,10 @@ The ``translation:update`` command looks for missing translations in:
 
     If you want to see the missing translation strings without actually updating
     the translation files, remove the ``--force`` option from the command above.
+    
+.. note::
+
+    If you want to see translation output strings sorted alphabetically, add ``--sort=asc`` option. Or you can use ``--sort=desc`` for reverse order.
```
Comment (OskarStark) [inline translation.rst]:
```suggestion
    If you want to see translation output strings sorted alphabetically, add ``--sort=asc`` or ``--sort=desc`` option.```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Just a minor comment

File: translation.rst (hunk 1 above)
Comment (k0d3r1s) [reply translation.rst]: i wanted to do that at first but i think that my version reads better. am i wrong?

Comment (OskarStark) [reply translation.rst]: Lets ask @javiereguiluz for another opinion 😃

**Comment by javiereguiluz** [comment]
Dāvis, thanks for contributing this feature and for mentioning it in the docs. It's now merged!

**Pull request merged**